api/utils/kubernetes.go

```diff
@@ -121,6 +126,9 @@ func getInClusterClientConfig(options []RestClientConfigOption) *restclient.Conf
 			log.Fatal().Err(err).Msg("getClusterConfig InClusterConfig")
 		}
 	}
+	config.Wrap(logs.Logger(func(e *zerolog.Event) {
```

Comment:
Should we make roundtripper logging configurable via options?

Reply:
Can be activated by setting log level to trace :)